core/src/com/theironyard/MyGdxGame.java

```diff
@@ -0,0 +1,113 @@
+package com.theironyard;
+
+import com.badlogic.gdx.ApplicationAdapter;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+
+public class MyGdxGame extends ApplicationAdapter {
+	SpriteBatch batch;
+	TextureRegion stand, jump, up, down, right, left;
+    float x, y = 0;
+    public static final float MAX_JUMP_VELOCITY = 200;
+    public static final float GRAVITY = -40;
+    private String faceDirection = null;
+
+	
+	@Override
+	public void create () {
+		batch = new SpriteBatch();
+        Texture tiles = new Texture("tiles.png");
+        TextureRegion[][] grid = TextureRegion.split(tiles, 16, 16);
+        down = grid[6][0];
+        up = grid[6][1];
+        right = grid[6][3];
+        left = new TextureRegion(right);
+        left.flip(true, false);
+
+        jump = grid[7][2];
+        stand = grid[6][2];
+	}
+
+	@Override
+	public void render () {
+        move();
+		Gdx.gl.glClearColor(0, 0.9f, 0.9f, 1);
+		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
+
+
+        batch.begin();
+        batch.draw(stand, x, y);
+        if(faceDirection == "up"){
+            batch.draw(up, x, y);
+        } else if(faceDirection == "down"){
+            batch.draw(down, x ,y);
+        } else if(faceDirection == "right"){
+            batch.draw(right, x, y);
+        } else if(faceDirection == "left"){
+            batch.draw(left, x, y);
+        }
+
+		batch.end();
+	}
+
+    public void move() {
+        if(Gdx.input.isKeyPressed(Input.Keys.J)) {
+            y = MAX_JUMP_VELOCITY + GRAVITY;
+        }
+
+        if (Gdx.input.isKeyPressed(Input.Keys.UP)) {
+            y++;
+            faceDirection = "up";
```

Comment:
Comparing strings is always a tricky thing for systems to do.  I would recommend not using random strings as values to switch on like this.  One way is to use an `enum` as this is a case of what they were used for.  Alternatively, you can create 4 constant variables that are each integers.

```java
public static final int UP = 1;
public static final int DOWN = 2;
public static final int LEFT = 3;
public static final int RIGHT = 4;
```
This not only is more efficient, it makes the code more readable and takes up less space.  It will look like this
`else if (faceDirection == LEFT)`.  Much more readable and prevents spelling errors as you use the variables.